docs/design/coreclr/botr/readytorun-format.md

```diff
@@ -578,6 +608,33 @@ Number of assemblies stored in the manifest metadata is equal to the number of M
 MVID records are used at runtime to verify that the assemblies loaded match those referenced by the
 manifest metadata representing the versioning bubble.
 
+## ReadyToRunSectionType.CrossModuleInlineInfo (v6.2+)
+The inlining information section captures what methods got inlined into other methods. It consists of a single _Native Format Hashtable_ (described below).
+
+The entries in the hashtable are lists of inliners for each inlinee. One entry in the hashtable corresponds to one inlinee. The hashtable is hashed the version resilient hashcode of the uninstantiated methoddef inlinee.
+
+The entry of the hashtable is a counted sequence of compressed unsigned integers which begins with an InlineeIndex which has flags which then defines how a sequence of inliners may be read from the file.
+
+* InlineeIndex
+  * Index with 2 flags field in lowest 2 bits to define the inlinee
+    - If flags & 1 == 0 then index is a MethodDef RID, and if the module is a composite image, a module index of the method follows
```

Comment:
Nit - assuming this is supposed to be a C++ / C# pseudo-code sequence, I believe the bitwise and operator would need to be parenthesized. Not sure if it's worth addressing.

Reply:
Done.